vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -834,6 +834,21 @@ testPam() {
   return $retval
 }
 
+testPrefetchScripts() {
+  local test="testPrefetchScripts"
+  local cni_prefetch_script="/opt/azure/containers/cni-prefetch.sh"
```

Comment:
nit: consider adding path suffix to var name to make more sense in the echo statement that follows, maybe "cni_prefetch_script_path"